house/views.py

```diff
@@ -13,7 +19,12 @@ def house_login(request):
 
 
 def house_view(request, house_id):
-    raise NotImplementedError
+    house = get_object_or_404(House, pk=house_id)
+    expenses_list = Expenses.objects.filter(house_name=house)
```

Comment:
Hey, I think there is a better way to do that in Django `Expenses.objects.filter(house_name=house)` 
Please take a look here: https://docs.djangoproject.com/en/4.0/topics/db/queries/#following-relationships-backward
Or here:  https://stackoverflow.com/questions/34184046/how-to-get-all-objects-referenced-as-foreignkey-from-given-field-in-a-module-in

Reply:
for now, I'll leave it as it is, 
I'll take a look again
thanks
@ronyyosef